segments/root.py

```diff
@@ -1,7 +1,7 @@
 def add_root_indicator_segment():
     root_indicators = {
         'bash': ' \\$ ',
-        'zsh': ' \\$ ',
+        'zsh': ' $ ',
```

Comment:
What's up with this change? If I don't hear back from you I will merge this, but undo this change.